### PR TITLE
TDR-3129 Create collector ECR repository for x-ray

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: root.tf [hunk 1]
```diff
@@ -261,8 +261,21 @@ module "ecr_consignment_api_repository" {
...
+    staging_account = data.aws_ssm_parameter.staging_account_number.value,
+    prod_account    = data.aws_ssm_parameter.prod_account_number.value
+  }
+  common_tags = local.common_tags
+}
+
+module "ecr_collector_repository" {
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "transfer-frontend-collector"
+  image_source_url = "https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/Dockerfile-collector"
```
Comment (TomJKing) [inline root.tf:278]: The `image_source_url` value should point to the x-ray repo Docker file: `https://github.com/nationalarchives/tdr-xray-logging/blob/main/Dockerfile`

File: root.tf [hunk 2]
```diff
@@ -261,8 +261,21 @@ module "ecr_consignment_api_repository" {
...
   policy_name      = "transfer_frontend_policy"
```
Comment (TomJKing) [inline root.tf:279]: Will probably have to create a new policy with the correct permissions, but this one will do for now, until we try all this out.

File: root.tf [hunk 3]
```diff
@@ -261,8 +261,21 @@ module "ecr_consignment_api_repository" {
 
 module "ecr_transfer_frontend_repository" {
```
[the remark this answers is not in the document]
Comment (thanhz) [reply root.tf]: Oops didn't mean to do that :man_facepalming:

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks. I've already deployed these changes, so the new ECR repository is on AWS.

**Pull request merged**